NEWS.md

```diff
@@ -1,5 +1,10 @@
 # rsconnect (development version)
 
+* `deployApp()` and friends will now respect packrat and renv lock files,
+  if present. If you don't want to use these lockfiles, and instead return
```

Comment:
Any reason we want to support both packrat and renv? Are there reasons for folks to not migrate?

Reply:
It's trivial (i.e. a no-op) to support packrat, so why not?